ClientApp/src/app/+chat/chat.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit, Inject } from '@angular/core';
-import { HubConnection } from '@aspnet/signalr';
+import { HubConnection } from '@aspnet/signalr-client';
```

Comment:
@thodoris did you also replace  "@aspnet/signalr" by  "@aspnet/signalr-client" in package.json? The package is deprecated https://www.npmjs.com/package/@aspnet/signalr-client

Reply:
@kmiskiewicz ooops my mistake. I missed the change in package.json